packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -151,7 +151,7 @@ const TradeParamsModal = observer(({ is_open, toggleModal, tab_index }: TTradePa
                 is_open={is_open}
                 header={<div />}
                 toggleModal={toggleModal}
-                height={is_vanilla ? '53.8rem' : 'auto'}
+                height='53.8rem'
```

Comment:
`height` and `width` here are params. If they are not passed, then 'auto' will be applied